3ein39/BookEra
# Add Toggle Button Feature

### Summary:
This pull request introduces a new feature to the project: a toggle button. The changes made in the commit with the hash `e1790b4` include the implementation of the toggle button functionality.

### Changes:
- Added toggle button component
- Integrated toggle functionality into the main codebase

### Purpose:
The toggle button enhances the user experience by providing a convenient way to switch between states or enable/disable specific features. This addition aligns with the project's goals of improving usability and functionality.

### How to Test:
1. Clone this branch locally
2. Run the application
3. Navigate to the relevant section where the toggle button is expected
4. Interact with the toggle button and observe the corresponding changes
